src/style.css

```
@@ -1,3 +1,80 @@
+
 body {
   margin: 0;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+
+  --active-color: #00ACDC;
+}
+
+header{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.logo_container{
+  display: flex;
+}
+
+.nav_container{
+  position: relative;
+  height: 100%;
+}
+
+.nav_container__list{
+  display: flex;
+  align-items: center;
+  padding: 0;
+  height: 100%;
+  margin: 0;
+}
+
+.list_item{
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0 10px;
+  height: 100%;
+}
+
+.list_item__link{
```

```suggestion
.list_item__link {
```
Add space before bracket, fix in all places

---

Still not fixed, and in other places too